libs/client-sdk/tests/data_source_event_handler_test.cpp

```diff
@@ -90,7 +90,7 @@ TEST(StreamingDataHandlerTests, BadSchemaPut) {
         ContextBuilder().Kind("user", "user-key").Build(), test_handler, logger,
         status_manager);
 
-    auto res = stream_handler.HandleMessage("put", "{\"potato\": {}}");
+    auto res = stream_handler.HandleMessage("put", "{\"potato\": 3}");
```

Comment:
Can I get an explainer on this one?

Reply:
Reverted. 

This was due to some confusion on my part - I intended to make `{}` be a valid message (viewing it as an implicit value of `0` for `version`, and `null` for `value`.) 

But I implemented that logic incorrectly and made it a valid deserialization because I was treating it as empty/omitted. 

That's a backwards way of thinking about it: "it's an empty object, therefore treat it as empty/omitted" vs "it's omitted/omitted, therefore treat it as an empty object" - the former is wrong, the latter is what I'm doing in the later Flag Model PR.